test/unitTests/observers/telemetryObserver.test.ts

```diff
@@ -24,6 +27,15 @@ describe('TelemetryObserver', () => {
             sentEventName = eventName;
             sentEventProperties = properties;
             sentEventMeasurements = measurements;
+        },
+        sendTelemetryMetric(
```

Comment:
I am just wondering what we are trying to test here.
With this setup, the input would be set to the predefined fields, 
so are they always the same once the `sendTelemetryMetric` method has been invoked?
Can we simplify to verify just 1 field, let's say `sentMetricName`?
I guess what we really want to verify is, `sendTelemetryMetric` method has been invoked during `observer.eventHandler(event)`, right?

Reply:
IMO, what should be verified is `sendTelemetryMetric ` has been invoked with right parameters

Reply:
ok